section2/reflection.md

```diff
@@ -1,13 +1,19 @@
 ## Section 2 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+  I currently try to wait to see if I understand later which works against me sometimes because I wait too long.
```

Comment:
I work in a very similar way, and found pre-learning essential at Turing. Try working through the lessons for the next day ahead of time so you have time to ponder some questions and either see if they get answered in class or ask for clarification.